website/thaliawebsite/settings/settings.py

```diff
@@ -329,6 +329,9 @@
 THALIA_PAY_ENABLED_PAYMENT_METHOD = (
     os.environ.get("THALIA_PAY_ENABLED", "False") == "True"
 )
+THALIA_PAY_FOR_NEW_MEMBERS = (
+    os.environ.get("THALIA_PAY_FOR_NEW_MEMBERS", "True") == "True"
```

Comment:
```suggestion
    os.environ.get("THALIA_PAY_FOR_NEW_MEMBERS", "False") == "True"
```

Reply:
Potato potato :)

Let's not forget this in the release

Reply:
> Let's not forget this in the release

Even if we forget it, but we don't change `THALIA_PAY_ENABLED` its not a problem